content/en/threat-model.md

```diff
@@ -0,0 +1,141 @@
+
+This means that secure distribution of Sigstore key material is paramount for security—if that process can be intercepted, we can consider all of the infrastructure effectively compromised.
+
+To deal with this, we recommend the use of a strong root of trust for distributing the key material. Once that root of trust is established, all additional key material can flow through the root of trust. Specifically, we recommend using The Update Framework (TUF) to distribute this key material, as is done by the public good instance. This supports features like:
+
+* Threshold signing by root-of-trust keys (the root of trust can comprise 5 keys, and signatures from 3 of 5 are needed to add new top-level key material).
+* Root keys are offline.
+* Rotation of both the root keys and other key material.
+* Revocation: compromised material can be marked as such, with an indication of the compromise time to allow continued verification of legitimate signatures from before the compromise.
+* Freshness: if an attacker tries to use an old key by preventing the victim from getting updates to key material (or serving old key material), the victim will detect this.
+
+The Sigstore public good instance has a [root of trust](https://github.com/sigstore/root-signing) based on TUF that can be publicly audited, with geographically- and organizationally-distributed root key holders.
+
+### Policy considerations
+The security guarantees that Sigstore provides are useful but relatively minimal: it can show you that a signature came from someone controlling a specific digital identity, but not whether you should trust that identity. Critically, [not everything that’s signed is secure](https://blog.sigstore.dev/signatus-ergo-securus-who-can-sign-what-with-tuf-and-sigstore-ea4d3d84b8b6): when verifying software, you need a policy for knowing whom to trust.
+
+That policy should cover:
+
+* How do you identify software artifacts (e.g., by their name)?
+* Which identities are permitted to sign a given software artifact?
+* Which identity providers can those identities come from?
+    * Do the identity providers perform identity verification like “know-your-customer?”
+    * Do they handle account compromise and recovery?
+* Do you permit the use of public verification keys for identities, or must identities have an associated identity provider?
+* Which Sigstore instance(s) do you trust, and how do you retrieve the key material for those instances?
+* How do you handle revocation?
+
+The answers will be different in different settings. For instance, a small organization may be able to get away with requiring signatures from one fixed party, with a blocklist for revoked artifacts. A large package repository may need to manage signing identities that change frequently over time. We hope to provide more detailed guidance for a variety of settings in future documentation.
```

Comment:
Maybe instead of "get away with" it can be more positive phrasing, like "a small organization with specific needs may only require signatures from one fixed party ... "